src/server/parser.js

```diff
@@ -44,7 +44,7 @@ function getMultipleErrors(errors) {
   errArray = [];
 
   for (key in errors) {
-    if (errors.hasOwnProperty(key)) {
+    if (Object.prototype.hasOwnProperty.call(errors, key)) {
```

Comment:
Can you add a comment describing why this is necessary? I can see someone coming along and "fixing" this to use `errors.hasOwnPropery(key)` if they didn't know better.